scripts/travis/install_gridtools.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+##===-------------------------------------------------------------------------------*- bash -*-===##
+##                         _       _
+##                        | |     | |
+##                    __ _| |_ ___| | __ _ _ __   __ _ 
+##                   / _` | __/ __| |/ _` | '_ \ / _` |
+##                  | (_| | || (__| | (_| | | | | (_| |
+##                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+##                    __/ |                       __/ |
+##                   |___/                       |___/
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+# @brief Install the GridTools libraries
+#
+# @param $1   Install directory
+# @param $2   GridTools version triple (X.Y.Z)
+# @param $3   GridTools components to build (',' separated)
+function install_gridtools() {
+  pushd $(pwd)
+  local start_time=$(date +%s)
+
+  if [[ $# -lt 2 ]]; then
+    fatal_error "argument mistmatch: ${FUNCNAME[0]} <install_prefix> <version>"
+  fi
+
+  local install_dir=$1
+  shift
+  local gridtools_version=$1
```

Comment:
maybe rename this to `gridtools_branch` as it reflects more the intended usage

Reply:
done